modules/light-clients/07-celestia/client_state.go

```diff
@@ -20,26 +20,6 @@ func (*ClientState) ClientType() string {
 	return ModuleName
 }
 
-// GetLatestHeight implements exported.ClientState.
-func (cs *ClientState) GetLatestHeight() exported.Height {
```

Comment:
I removed this functions from `ClientState` and call instead `.BaseClient` directly in the light client module. I think it is cleaner that way, but the drawback that the backport to v8.2.x will require more work.

Reply:
Will we need to backport this feature?

Reply:
Yes, we will have to backport these changes to the release branch of the DA light client that is compatible with ibc-go v8.3.x (and that version doesn't include the 02-client routing refactor).

Reply:
It might be less overhead to take whats on the target branch now and separate that to a different branch for v8.3.x before merging this PR. It's no harm at least to replicate the feat/celestia-da-client before this PR is merged in case that might be helpful.